pkg/auth/authenticator_test.go

```diff
@@ -436,6 +436,61 @@ func TestTestAccessBadIPList(t *testing.T) {
 	assert.Empty(t, extraHeaders, "Expected no extra headers for invalid IP")
 }
 
+func TestTestAccessLimited(t *testing.T) {
+	authenticator := &Authenticator{
+		accessCache:   &AccessCache{},
+		servicesCache: &ServicesCache{},
+	}
+
+	tokenEntry := AccessCacheEntry{
+		AccessToken: cerberusv1alpha1.AccessToken{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "valid-token",
+			},
+			Spec: cerberusv1alpha1.AccessTokenSpec{
+				Priority: 50,
+			},
+		},
+		allowedServices: map[string]struct{}{
+			"SampleWebService": {},
+		},
+	}
+	(*authenticator.accessCache)["valid-token"] = tokenEntry
+
+	// Assuming an IP not in the allow list
```

Comment:
what do we have to do with IP here?

Reply:
It was mistake and it's replaced with a meaningful comment.